src/api_server.py

```diff
@@ -115,8 +53,9 @@ def startup_event():
     logger.info("Starting API server")
 
     classifier = BenefitsAndDrawbacksExtractor(
-        model_name=cnf.model_name,
+        model_qa=cnf.model_qa,
         batch_size=cnf.batch_size,
+        model_seq2seq=cnf.model_seq2seq,
     )
```

Comment:
`BenefitsAndDrawbacksExtractor` has multiple responsibilities. It needs to handle both the QA model and the Seq2seq model. Suggestion: create a separate class for the seq2seq model.

Reply:
See example: https://github.com/choosistant/algorithms/pull/16#discussion_r992268238